app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception, prepend: true
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+    def configure_permitted_parameters
+      devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:first_name, :last_name, :birthdate, :languages, :contact_number, :account_type, :email, :password)}
+
+      devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:first_name, :last_name, :birthdate, :languages, :contact_number, :account_type, :email, :password, :current_password)}
```

Comment:
Remove `email`, `password`, and `current_password`. These are already supported by Devise.

Reply:
this PR isn't using Devise just yet, the Devise implementation was implemented in PR #4

Reply:
It seems like you have added Devise to this PR already with your commit: "converted to use Devise Gem".
Please make this PR work with Devise before adding any changes to the others.